testing/openttd/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Adrian Siekierka <kontakt@asie.pl>
+# Maintainer: Adrian Siekierka <kontakt@asie.pl>
+pkgname=openttd
+pkgver=1.6.1
+pkgrel=0
+pkgdesc="Open source simulation game based upon Transport Tycoon Deluxe"
+url="http://openttd.org/"
+arch="all"
+license="GPL2"
+depends=""
```

Comment:
I think that you’ve reversed dependencies – openttd should depend on `openttd-opensfx` and `openttd-opengfx`, not vice versa.